src/commands/moderation/ban.ts

```diff
@@ -37,50 +38,61 @@ export default class Ban {
       ].join("\n"))
       .setThumbnail(user.displayAvatarURL())
-      .setAuthor({ name: `• ${user.username}`, iconURL: user.displayAvatarURL() })
       .setFooter({ text: `User ID: ${user.id}` })
       .setColor(genColor(100));
 
     const embedDM = new EmbedBuilder()
+      .setAuthor({ name: `• ${user.username}`, iconURL: user.displayAvatarURL() })
       .setTitle(`🔨 • You were banned`)
       .setDescription([
         `**Moderator**: ${interaction.user.username}`,
         `**Reason**: ${reason ?? "No reason provided"}`
       ].join("\n"))
       .setThumbnail(user.displayAvatarURL())
-      .setAuthor({ name: `• ${user.username}`, iconURL: user.displayAvatarURL() })
       .setFooter({ text: `User ID: ${user.id}` })
       .setColor(genColor(0));
 
-    if (!member.permissions.has(PermissionsBitField.Flags.BanMembers))
-      return await interaction.followUp({ embeds: [errorEmbed("You need the **Ban Members** permission to execute this command.")] });
-    if (selectedMember === member)
-      return await interaction.followUp({ embeds: [errorEmbed("You can't ban yourself")] });
-    if (selectedMember.user.id === interaction.client.user.id)
-      return await interaction.followUp({ embeds: [errorEmbed("You can't ban Nebula.")] });
-    if (!selectedMember.manageable)
-      return await interaction.followUp({ embeds: [errorEmbed(`You can't ban ${name}, because they have a higher role position than Nebula.`)] });
-    if (member.roles.highest.position < selectedMember.roles.highest.position)
-      return await interaction.followUp({ embeds: [errorEmbed(`You can't ban ${name}, because they have a higher role position than you.`)] });
+    if (!member.permissions.has(PermissionsBitField.Flags.BanMembers)) return await interaction.reply({
+      embeds: [errorEmbed("You need the **Ban Members** permission to execute this command.")]
+    });
+
+    if (selectedMember === member) return await interaction.reply({
+      embeds: [errorEmbed("You can't ban yourself.")]
+    });
+
+    if (selectedMember.user.id === interaction.client.user.id) return await interaction.reply({
+      embeds: [errorEmbed("You can't ban Nebula.")]
+    });
+
+    if (!selectedMember.manageable) return await interaction.reply({
+      embeds: [errorEmbed(`You can't ban ${name}, because they have a higher role position than Nebula.`)]
+    });
+
+    if (member.roles.highest.position < selectedMember.roles.highest.position) return await interaction.reply({
+      embeds: [errorEmbed(`You can't ban ${name}, because they have a higher role position than you.`)]
+    });
 
     const db = this.db;
     const settingsTable = await getSettingsTable(db);
-    const logChannel = await settingsTable?.get(`${interaction.guild.id}.logChannel`).then(
-      (channel: string | null) => channel ?? null
-    ).catch(() => null);
+    const logChannel = await settingsTable
+      ?.get(`${interaction.guild.id}.logChannel`)
```

Comment:
```suggestion
    const logChannel = await settingsTable?
      .get(`${interaction.guild.id}.logChannel`)
```
That character belongs there I think ;)

Reply:
i reckon it errored so it needed `?` to be there

Reply:
Oh sad, doesn't look that clean